modules/admin_manual/pages/release_notes.adoc

```diff
@@ -434,6 +434,11 @@ Version 10.1 delivers all the prerequisites to be compatible with the Microsoft
 
 ownCloud Server 10.1 introduces WebDAV Locks that allow clients to lock and unlock resources to prevent other users from making changes. The feature has been implemented as a prerequisite for manual file locking and MS Office Online Server compatibility. In the current state, file locking is only available via API. Users can recognize locked files via the "lock" icon in the file list. Additionally a lock owner (the user who locked the file) can manually unlock them via the "Locks" tab in the right sidebar. The "Locks" tab will only appear for files that have active locks.
 
+=== Foreign Keys in Database
+
+Please note that foreign keys have been added with the :xref:webdav-locks[WebDAV Locks] feature.
+This is the first time ownCloud implements foreign keys.
```

Comment:
What exactly are the implications for each supported database?
e.g. in MySQL, MariaDb, PostgreSQL, Oracle is it possible to have "foreign keys support" off? If someone has done that, does something bad happen? Do they have to make sure to set "foreign keys support" on?

What happens with sqlite?

After knowing these answers, then it would be possible to write something that gives an administrator some clue about what to check.

Reply:
Thanks for jumping in so quickly, @phil-davis.

Reply:
I added information about foreign key support. From research, the only database that could be affected is SQLite. The other databases all support foreign keys and appear to have that support baked in and enabled by default.